.github/workflows/ci.yaml

```diff
@@ -142,3 +141,24 @@ jobs:
           --exclude rosetta-server-ethereum \
           --exclude rosetta-server-polkadot \
           --exclude rosetta-client
+
+    - name: Cleanup Docker
+      run: ./scripts/reset_docker.sh
+  
+  #Running arbitrum nitro-testnode
+  arbitrum-nitro-testnode-script:
+    needs: [rustfmt, test-client]
+    runs-on: self-hosted
+    steps:
+    - name: Checkout code
+      run: git clone -b release --recurse-submodules https://github.com/ManojJiSharma/nitro-testnode.git
```

Comment:
This must be added as a submodule dependency to this repo, so when we clone it, we also clone this repo, this is safer because submodules links to a specific commit.

Instructions on how to add a submodule:
https://git-scm.com/book/en/v2/Git-Tools-Submodules

Reply:
this doc also suggested to do in the way I have done.